charts/posit-chronicle/values.yaml

```diff
@@ -78,6 +78,15 @@ pod:
     fsGroup: 1000
     fsGroupChangePolicy: "OnRootMismatch"
 
+resources:
+  requests:
+    memory: "2Gi"
+    cpu: "1000m"
+    ephemeralStorage: "100Mi"
```

Comment:
I don't have the whole context for these changes, but we should be sure that setting minimums and limits on CPU and ephemeral storage is actually necessary. I think including the option for users is nice though.